### PR TITLE
Added support for Materialized Views on postgresql

### PR DESCRIPTION
**Pull request by bakjos**

### CODE REVIEW

**Comment by facebook-github-bot-0** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

**Comment by facebook-github-bot-5** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

**Comment by electrum** [comment]
Can you add a test for this in `TestPostgreSqlIntegrationSmokeTest`? You can use the `execute()` method to create the materialized view in PostgreSQL, then use `assertQuery()` to verify that the connector can read from it.

File: presto-postgresql/src/main/java/com/facebook/presto/plugin/postgresql/PostgreSqlClient.java [hunk 1]
```diff
@@ -62,5 +64,18 @@ public PreparedStatement getPreparedStatement(Connection connection, String sql)
         PreparedStatement statement = connection.prepareStatement(sql);
         statement.setFetchSize(1000);
         return statement;
-    }
+   }
+
+   @Override
```
Comment (electrum) [inline presto-postgresql/src/main/java/com/facebook/presto/plugin/postgresql/PostgreSqlClient.java]: The indentation is off here. It should be indented 4 spaces. This is the code style: https://github.com/airlift/codestyle

**Comment by martint** [comment]
Merged, thanks!

**Pull request closed without merging**